test-data/ej-fra/ej-fra-translated.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version='1.0' encoding='utf-8'?>
+<TEI>
+    <!-- To exclude any element from alignment, add the do-not-align="true" attribute to
+         it, e.g., <p do-not-align="true">...</p>, or
+         <s>Some text <foo do-not-align="true">do not align this</foo> more text</s> -->
+    <text xml:lang="fra" id="t0">
+        <body id="t0b0">
+            <div type="page" id="t0b0d0" class="two-column-layout-page">
+              <graphic url="avatar.png" />
```

Comment:
I love the avatar of Eric! 😂

Reply:
Sorry for raining on your parade, @deltork and @eddieantonio, but this image requires attribution whenever it is used, or else it has to be purchased, so we should remove it and use something else. I found its provenance here https://www.flaticon.com/free-icon/captain_190659?related_id=190659&origin=pack or here https://www.iconfinder.com/search/?q=services. Sites that use it free must have proper attribution according to the first link, and both links say you have to purchase it to use it without attribution.

Reply:
And of course the issue is that our WebC doesn't support displaying attributions, and we don't really want to have to do that now, or buy the rights to the icon...